test_scripts/UpdateVideoStreamingCapabilities/001_1_GetCapabilities_with_addVSC.lua

```diff
@@ -0,0 +1,34 @@
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0296-Update-video-streaming-capabilities-during-ignition-cycle.md
+--
+-- Description: SDL applies the videoStreamingCapability with additionalVideoStreamingCapabilities parameter
+--  received from HMI in UI.GetCapabilities response
+--
+-- Preconditions:
+-- 1. SDL and HMI are started
+
```

Comment:
```suggestion
--
```
I would propose to comment out this line. This comment relates to next scripts too.

Reply:
Done (see 38967ac)